examples/mqtt.rs

```diff
@@ -17,11 +18,16 @@ struct Settings {
 
 async fn mqtt_client() {
     // Construct a Minimq client to the broker for publishing requests.
-    let mut mqtt: Minimq<_, 256> =
-        Minimq::new("127.0.0.1".parse().unwrap(), "tester", Stack::default()).unwrap();
+    let mut mqtt: Minimq<_, _, 256, 3> = Minimq::new(
```

Comment:
Do we need more than 1 (or even 0?) in flight QoS messages for miniconf?

```suggestion
    let mut mqtt: Minimq<_, _, 256, 0> = Minimq::new(
```

Reply:
We cannot use a size of zero because of the underlying storage contains in minimq, but we can use zero.